maintainers/maintainer-list.nix

```diff
@@ -12820,6 +12820,11 @@
     github = "nitsky";
     githubId = 492793;
   };
+  nix-julia = {
```

Comment:
i think the `-` is ignored for sorting so `nix-julia` needs to come after `nixinator`